android/assets/jsons/translations/template.properties

```diff
@@ -1277,6 +1278,8 @@ Status ￬ =
 Must be next to [terrain] = 
 Must be on [terrain] = 
 +[amount]% vs [unitType] = 
++[amount]% Strenght vs [unitType] = 
++[amount]% Strenght vs [unitType] units = 
```

Comment:
These uniques are deprecated, so not having translations for them isn't a problem imo